db/index.js

```diff
@@ -48,19 +48,28 @@ class BarcodesDb {
 
   /**
    * initInsert()
-   * Insert the first value into the database if it's not already there. This is
-   * to set from where new barcodes will be created.
+   * Insert the first values into the database if they are not already there.
+   * This is to set from where new barcodes will be created based on p-types.
    */
   async initInsert() {
     const text = "INSERT INTO barcodes (barcode, used) VALUES ($1, $2);";
-    const values = ["28888855432452", "true"];
+    const barcodes = ["28888855432452", "25555001345283"];
```

Comment:
This is the main part and introduction of the new barcode. The insertion into the db is done in a `Promise.all` to make sure it successfully or unsuccessfully runs for both barcodes.